apps/openmw/mwmechanics/aiwander.hpp

```diff
@@ -63,6 +63,13 @@ namespace MWMechanics
                 Greet_InProgress,
                 Greet_Done
             };
+
+            enum WanderState {
+                Wander_ChooseAction,
+                Wander_IdleNow,
+                Wander_MoveNow,
+                Wander_Walking,
```

Comment:
apps/openmw/mwscript/../mwmechanics/aiwander.hpp:71:31: warning: comma at end of enumerator list [-Wpedantic]
                 Wander_Walking,
                               ^